src/ui/server/controllers/insurance/check-your-answers/export-contract/index.ts

```diff
@@ -67,7 +67,7 @@ export const get = async (req: Request, res: Response) => {
 
     const countries = await api.keystone.countries.getAll();
 
-    const summaryList = exportContractSummaryLists(exportContract, totalContractValueOverThreshold, referenceNumber, countries, checkAndChange);
+    const summaryList = exportContractSummaryLists(exportContract, totalContractValueOverThreshold, migratedV1toV2, referenceNumber, countries, checkAndChange);
```

Comment:
with so many variables, would this be better in an object?

Reply:
Yes, I thought the same - want to update all summary list functions to be object structured. Really keen to do this but it's not quick and would double/triple the size of this PR.

We need to get these changes in ASAP for data migration testing, before public beta.

Can tackle this and all other summary list functions to be object structured in `main-iterations`

Reply:
Created EMS-3748 for this